native/src/components/SimpleImage.tsx

```diff
@@ -30,12 +30,14 @@ const SimpleImage = ({
   resizeMode = 'contain',
   specifyAspectRatio = false,
 }: SimpleImageProps): ReactElement => {
-  const [aspectRatio, setAspectRatio] = useState<number | undefined>(undefined)
-
-  useEffect(() => {
+  const aspectRatio = useMemo(() => {
```

Comment:
`useMemo` is resolved before the first render while `useEffect` is only run after the first render. This leads to weird test behavior. Also useMemo seems to be the right choice here, as we just assign a value.